spec/hscode/config/bootstrap_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+describe Bootstrap::Config do
+  describe '#get' do
+    let(:root) { File.expand_path('../../../../', __FILE__) }
+
+    it 'gets all environment variables' do
+      File.open("#{root}/config/env.yaml", 'w+') do |f|
```

Comment:
Yes, I'd clean this up later today.

Actual configs could become overwritten if we have one. I just think it's good we test everything in this starting phase before it gets complicated.

If we can find a better way to test this, would be glad to implement it straightaway.